imports/ui/routes/welcome/index.js

```diff
@@ -15,7 +15,7 @@ const piStyle = {
 
 const Welcome = ({ push }) => (
   <Column alignItems="center" justifyContent="center" style={{ height: '100%' }}>
-    <p>Hi, { Meteor.user().username }!</p>
+    <p>Hi, { Meteor.user() && Meteor.user().username }!</p>
```

Comment:
Maybe we should include the comma in a ternary, otherwise if there's no username we'll have a weird dangling comma.